### PR TITLE
sprinkle related links on wip pages

### LINKED ISSUE #78: [CONTENT] Add related links to under-construction pages
For under-construction pages (particularly high-value ones), please add any related links you can to the frontmatter. This has already been done in `docs/tutorial/using-nfts-in-games.md` as an example:

```
related:
  'Guide to NFTs, DeFi, and Gaming (Hackernoon)': https://hackernoon.com/everything-you-need-to-know-about-nfts-defi-and-gaming-rbm335n
  'NFTs: The Next Gaming Revolution? (Ledger)': https://www.ledger.com/academy/nfts-the-next-gaming-revolution
```

Example in situ: https://nft-school-staging.on.fleek.co/tutorial/using-nfts-in-games/

Thank you!

### PR DESCRIPTION
**Pull request by yusefnapora**
Closes #78

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/blockchains.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2019/11/08/crypto-glossary/
### `docs/concepts/distributed-web.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2019/11/08/crypto-glossary/
- Check of `docs/how-to/creating-nfts.md` came back clean!
### `docs/reference/example-apps-code-samples.md`:
- Consider using an m-dash if you do not want to join two words.
  ```
  ...ry Mint an NFT with IPFSThe Buidl Guidl - ethereum examples built with scaff...
                                             ^
  ```
### `docs/reference/featured-sites.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2021/04/02/nfts-readings-resources/
### `docs/tutorial/gallery-app.md`:
- Consider using an m-dash if you do not want to join two words.
  ```
  ...ces to try Fleek Crypto Museum DemoNFTE - easily embed NFTs into a web pageN...
                                             ^
  ```
- Consider using an m-dash if you do not want to join two words.
  ```
  ... embed NFTs into a web pageNifty Viewer - forkable ethereum example app
                                             ^
  ```
- Check of `docs/tutorial/minting-app.md` came back clean!

File: docs/tutorial/gallery-app.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 title: Building a gallery app 🚧
 description: Get a step-by-step overview of building an NFT gallery app in this developer-focused guide.
 issueUrl: https://github.com/protocol/nft-website/issues/42
+related:
+  'Fleek Crypto Museum Demo': https://docs.fleek.co/examples/storage/
```
Comment (jessicaschilling) [inline docs/tutorial/gallery-app.md:6]:
```suggestion
  'Fleek Crypto Museum demo': https://docs.fleek.co/examples/storage/
```

File: docs/concepts/blockchains.md [hunk 2]
```diff
@@ -2,7 +2,15 @@
 title: Blockchains and NFTs 🚧
 description: Understand how blockchains and NFTs work together in this developer-focused guide.
 issueUrl: https://github.com/protocol/nft-website/issues/27
+related:
+  'A16Z Crypto Glossary': https://a16z.com/2019/11/08/crypto-glossary/
```
Comment (jessicaschilling) [inline docs/concepts/blockchains.md:6]:
```suggestion
  'A16Z crypto glossary': https://a16z.com/2019/11/08/crypto-glossary/
```

File: docs/concepts/blockchains.md [hunk 3]
```diff
@@ -2,7 +2,15 @@
...
+  'etherum.org NFT Overview': https://ethereum.org/en/nft/
```
Comment (jessicaschilling) [inline docs/concepts/blockchains.md:7]:
```suggestion
  'NFT overview from ethereum.org': https://ethereum.org/en/nft/
```

File: docs/concepts/distributed-web.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 title: NFTs and the distributed web 🚧
 description: Learn the basics of the distributed web and how it is important to NFT best practices in this developer-focused guide.
 issueUrl: https://github.com/protocol/nft-website/issues/28
+related:
+  'A16Z Crypto Glossary': https://a16z.com/2019/11/08/crypto-glossary/
```
Comment (jessicaschilling) [inline docs/concepts/distributed-web.md:6]:
```suggestion
  'A16Z crypto glossary': https://a16z.com/2019/11/08/crypto-glossary/
```

File: docs/concepts/distributed-web.md [hunk 5]
```diff
@@ -2,6 +2,10 @@
...
+  'IPFS': https://ipfs.io
```
Comment (jessicaschilling) [inline docs/concepts/distributed-web.md:7]:
```suggestion
  'IPFS homepage': https://ipfs.io
```

File: docs/how-to/creating-nfts.md [hunk 6]
```diff
@@ -2,8 +2,14 @@
 title: Creating NFTs 🚧
 description: Learn the basics of creating NFTs in this developer-focused guide, including minting best practices and how to store NFTs persistently.
 issueUrl: https://github.com/protocol/nft-website/issues/43
+related:
+  'IPFS best practices for NFTs': https://docs.ipfs.io/how-to/best-practices-for-nft-data/
+  'How to create NFTs with Flow and IPFS': https://medium.com/pinata/how-to-create-nfts-like-nba-top-shot-with-flow-and-ipfs-701296944bf
+  'How to create an NFT Marketplace with Flow': https://medium.com/pinata/how-to-create-an-nft-marketplace-on-flow-with-ipfs-a162a1aeb426
```
Comment (jessicaschilling) [inline docs/how-to/creating-nfts.md:8]:
```suggestion
  'How to create an NFT marketplace with Flow': https://medium.com/pinata/how-to-create-an-nft-marketplace-on-flow-with-ipfs-a162a1aeb426
```

File: docs/reference/example-apps-code-samples.md [hunk 7]
```diff
@@ -2,6 +2,11 @@
 title: Example apps and code samples 🚧
 description: Example apps, code samples, and more for NFT developers.
 issueUrl: https://github.com/protocol/nft-website/issues/34
+related:
+  'Mint an NFT with IPFS': https://docs.ipfs.io/how-to/mint-nfts-with-ipfs/
+  'The Buidl Guidl - ethereum examples built with scaffold-eth': https://buidlguidl.com/
```
Comment (jessicaschilling) [inline docs/reference/example-apps-code-samples.md:7]:
```suggestion
  'The Buidl Guidl: Ethereum examples built with scaffold-eth': https://buidlguidl.com/
```

File: docs/reference/example-apps-code-samples.md [hunk 8]
```diff
@@ -2,6 +2,11 @@
...
+  'OpenZeppelin Docs': https://docs.openzeppelin.com/openzeppelin/
```
Comment (jessicaschilling) [inline docs/reference/example-apps-code-samples.md:8]:
```suggestion
  'OpenZeppelin docs': https://docs.openzeppelin.com/openzeppelin/
```

File: docs/reference/featured-sites.md [hunk 9]
```diff
@@ -2,6 +2,10 @@
 title: Featured NFT dev sites 🚧
 description: Helpful sites and other resources for NFT developers.
 issueUrl: https://github.com/protocol/nft-website/issues/33
+related:
+  'OpenZeppelin': https://openzeppelin.com/
+  'A16Z NFT Canon': https://a16z.com/2021/04/02/nfts-readings-resources/
```
Comment (jessicaschilling) [inline docs/reference/featured-sites.md:7]:
```suggestion
  'A16Z NFT canon': https://a16z.com/2021/04/02/nfts-readings-resources/
```

File: docs/reference/featured-sites.md [hunk 10]
```diff
@@ -2,6 +2,10 @@
...
+  'OpenSea NFT Bible': https://opensea.io/blog/guides/non-fungible-tokens/
```
Comment (jessicaschilling) [inline docs/reference/featured-sites.md:8]:
```suggestion
  'OpenSea NFT bible': https://opensea.io/blog/guides/non-fungible-tokens/
```

File: docs/tutorial/gallery-app.md [hunk 11]
```diff
@@ -2,6 +2,11 @@
...
+  'NFTE - easily embed NFTs into a web page': https://nfte.app/
```
Comment (jessicaschilling) [inline docs/tutorial/gallery-app.md:7]:
```suggestion
  'NFTE: Easily embed NFTs into a web page': https://nfte.app/
```

File: docs/tutorial/gallery-app.md [hunk 12]
```diff
@@ -2,6 +2,11 @@
...
+  'Nifty Viewer - forkable ethereum example app': https://github.com/austintgriffith/scaffold-eth/tree/nifty-viewer
```
Comment (jessicaschilling) [inline docs/tutorial/gallery-app.md:8]:
```suggestion
  'Nifty Viewer: A forkable Ethereum example app': https://github.com/austintgriffith/scaffold-eth/tree/nifty-viewer
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/blockchains.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2019/11/08/crypto-glossary/
### `docs/concepts/distributed-web.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2019/11/08/crypto-glossary/
- Check of `docs/how-to/creating-nfts.md` came back clean!
- Check of `docs/reference/example-apps-code-samples.md` came back clean!
### `docs/reference/featured-sites.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2021/04/02/nfts-readings-resources/
- Check of `docs/tutorial/gallery-app.md` came back clean!
- Check of `docs/tutorial/minting-app.md` came back clean!

File: docs/concepts/blockchains.md [hunk 13]
```diff
@@ -2,7 +2,15 @@
...
+  'NFTs on Tezos': https://assets.tqtezos.com/docs/token-contracts/fa2/2-fa2-nft-tutorial/
+
 ---
  # Blockchains and NFTs
 
-<ContentStatus />
\ No newline at end of file
+<ContentStatus />
+
+## Useful Links
```
Comment (jessicaschilling) [inline docs/concepts/blockchains.md:16]:
```suggestion
```

File: docs/how-to/creating-nfts.md [hunk 14]
```diff
@@ -2,8 +2,13 @@
...
+related:
+  'IPFS best practices for NFTs': https://docs.ipfs.io/how-to/best-practices-for-nft-data/
+  'How to create NFTs with Flow and IPFS': https://medium.com/pinata/how-to-create-nfts-like-nba-top-shot-with-flow-and-ipfs-701296944bf
+  'How to create an NFT marketplace with Flow': https://medium.com/pinata/how-to-create-an-nft-marketplace-on-flow-with-ipfs-a162a1aeb426
 ---
  # Creating NFTs
 
 <ContentStatus />
 
+## References
```
Comment (jessicaschilling) [inline docs/how-to/creating-nfts.md:14]:
```suggestion
```

**Review by jessicaschilling — APPROVED** [review]
@yusefnapora LGTM, thanks!
Just FYI: You don't need to add any headers to the markdown, just `related` in the frontmatter. If there's anything marked as `related` in the frontmatter, the "Other resources to try" header gets automatically generated 😊

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/concepts/blockchains.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2019/11/08/crypto-glossary/
### `docs/concepts/distributed-web.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2019/11/08/crypto-glossary/
- Check of `docs/how-to/creating-nfts.md` came back clean!
- Check of `docs/reference/example-apps-code-samples.md` came back clean!
### `docs/reference/featured-sites.md`:
- The following link returned HTTP status code 403 (Forbidden):
  https://a16z.com/2021/04/02/nfts-readings-resources/
- Check of `docs/tutorial/gallery-app.md` came back clean!
- Check of `docs/tutorial/minting-app.md` came back clean!

**Pull request merged**